NIMBUS-687 :: signature-touch-fix

haandyha opened the pull request:
Modified signature.component.ts file to recognize touch events and added test for touch recognition.

haandyha committed:
Update signature.component.ts

haandyha committed:
tests for touch events recognition

openanthem-admin commented:
You did it @haandyha!

Thank you for signing the Anthem Opensource Software CLA.

haandyha committed:
NIMBUS:43355 test for touch event recognition

haandyha committed:
CMDM/NIMBUS: 43355 - test for touch recognition

test checks to see if the registerCaptureEvents function was called with string values 'touchstart' and 'touchend'

haandyha committed:
Signature is not working on 2 in 1 systems #708

test checks if registerCaptureOnEvent() is called with string values 'touchstart' and 'touchend'

haandyha committed:
NIMBUS-687 :: Unable to sign with finger on 2-in-1 pc's

sandeepmantha reviewed (commented):
Can you please add a capture type so that touch is not applicable when the user chooses to go that route ?

haandyha committed:
NIMBUS-687 :: signature.component.ts changes

Added two capture types to toggle touch event recognition

haandyha committed:
NIMBUS-687 :: viewConfig.java changes

added two new capture types to make touch recognition configurable

sandeepmantha commented on nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java:
```diff
@@ -89,22 +83,19 @@
-	 * <p><b>When the decorated field is nested within a {@link Section}</b></p>
-	 * <p>Nested {@link AccordionTab} components will render nested fields that
-	 * are decorated with: <ul> <li>{@link CardDetail}</li>
-	 * <li>{@link CardDetailsGrid}</li> <li>{@link ButtonGroup}</li>
```
Expecting on the touch events added ?

Commit:
modified touch event recognition to be configurable
